<launch>
    <!-- Bring in global hardware parameters -->
    <rosparam command="load" file="$(find system_globals)/params/frames.yaml" />
    <rosparam command="load" file="$(find topic_messages)/params/topics.yaml" />

    <!-- This node interfaces with the ultrasonic range sensor server and publishes sensor data -->
    <node
        pkg="range_sensor_control"
        name="range_filter_node"
        type="range_filter_node"
        output="screen"
    >
        <param name="sensor_frame" value="seeker/ultrasonic"/>
        <param name="filter_clutter" value="true" />
        <param name="num_measurements" value="10" />
        <param name="beam_width_scale" value="0.3" />
        <param name="timeout" value="0.13" />
        <param name="measurement_range" value="0.01" />
    </node>
</launch>